2022-06-08 12:59:17,224 - ERROR - basePage.py[51]:用户名元素定位不到
2022-06-08 12:59:17,225 - ERROR - basePage.py[52]:Traceback (most recent call last):
  File "/Users/shenyuming/PycharmProjects/autotest_UI/common/basePage.py", line 45, in get_element
    return WebDriverWait(self.driver,ENV.WEB_DRIVER_WAIT_TIMEOUT,ENV.POLL_FREQUENCY_TIME).until(EC.visibility_of_element_located(locator))
  File "/usr/local/lib/python3.7/site-packages/selenium/webdriver/support/wait.py", line 89, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
0   chromedriver                        0x0000000104883da4 chromedriver + 2538916
1   chromedriver                        0x0000000104e19928 chromedriver + 8395048
2   chromedriver                        0x0000000104640e60 chromedriver + 167520
3   chromedriver                        0x000000010466fb48 chromedriver + 359240
4   chromedriver                        0x0000000104698740 chromedriver + 526144
5   chromedriver                        0x0000000104665158 chromedriver + 315736
6   chromedriver                        0x0000000104850cb0 chromedriver + 2329776
7   chromedriver                        0x0000000104861030 chromedriver + 2396208
8   chromedriver                        0x0000000104837f64 chromedriver + 2228068
9   chromedriver                        0x0000000104861fc4 chromedriver + 2400196
10  chromedriver                        0x0000000104847c5c chromedriver + 2292828
11  chromedriver                        0x00000001048787d8 chromedriver + 2492376
12  chromedriver                        0x000000010487894c chromedriver + 2492748
13  chromedriver                        0x0000000104888768 chromedriver + 2557800
14  libsystem_pthread.dylib             0x000000019c8de06c _pthread_start + 320
15  libsystem_pthread.dylib             0x000000019c8d8da0 thread_start + 8


2022-06-08 12:59:28,632 - ERROR - basePage.py[51]:用户名元素定位不到
2022-06-08 12:59:28,632 - ERROR - basePage.py[52]:Traceback (most recent call last):
  File "/Users/shenyuming/PycharmProjects/autotest_UI/common/basePage.py", line 45, in get_element
    return WebDriverWait(self.driver,ENV.WEB_DRIVER_WAIT_TIMEOUT,ENV.POLL_FREQUENCY_TIME).until(EC.visibility_of_element_located(locator))
  File "/usr/local/lib/python3.7/site-packages/selenium/webdriver/support/wait.py", line 89, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
0   chromedriver                        0x0000000104883da4 chromedriver + 2538916
1   chromedriver                        0x0000000104e19928 chromedriver + 8395048
2   chromedriver                        0x0000000104640e60 chromedriver + 167520
3   chromedriver                        0x000000010466fb48 chromedriver + 359240
4   chromedriver                        0x0000000104698740 chromedriver + 526144
5   chromedriver                        0x0000000104665158 chromedriver + 315736
6   chromedriver                        0x0000000104850cb0 chromedriver + 2329776
7   chromedriver                        0x0000000104861030 chromedriver + 2396208
8   chromedriver                        0x0000000104837f64 chromedriver + 2228068
9   chromedriver                        0x0000000104861fc4 chromedriver + 2400196
10  chromedriver                        0x0000000104847c5c chromedriver + 2292828
11  chromedriver                        0x00000001048787d8 chromedriver + 2492376
12  chromedriver                        0x000000010487894c chromedriver + 2492748
13  chromedriver                        0x0000000104888768 chromedriver + 2557800
14  libsystem_pthread.dylib             0x000000019c8de06c _pthread_start + 320
15  libsystem_pthread.dylib             0x000000019c8d8da0 thread_start + 8


2022-06-08 12:59:46,166 - ERROR - basePage.py[51]:用户名元素定位不到
2022-06-08 12:59:46,167 - ERROR - basePage.py[52]:Traceback (most recent call last):
  File "/Users/shenyuming/PycharmProjects/autotest_UI/common/basePage.py", line 45, in get_element
    return WebDriverWait(self.driver,ENV.WEB_DRIVER_WAIT_TIMEOUT,ENV.POLL_FREQUENCY_TIME).until(EC.visibility_of_element_located(locator))
  File "/usr/local/lib/python3.7/site-packages/selenium/webdriver/support/wait.py", line 89, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
0   chromedriver                        0x0000000104883da4 chromedriver + 2538916
1   chromedriver                        0x0000000104e19928 chromedriver + 8395048
2   chromedriver                        0x0000000104640e60 chromedriver + 167520
3   chromedriver                        0x000000010466fb48 chromedriver + 359240
4   chromedriver                        0x0000000104698740 chromedriver + 526144
5   chromedriver                        0x0000000104665158 chromedriver + 315736
6   chromedriver                        0x0000000104850cb0 chromedriver + 2329776
7   chromedriver                        0x0000000104861030 chromedriver + 2396208
8   chromedriver                        0x0000000104837f64 chromedriver + 2228068
9   chromedriver                        0x0000000104861fc4 chromedriver + 2400196
10  chromedriver                        0x0000000104847c5c chromedriver + 2292828
11  chromedriver                        0x00000001048787d8 chromedriver + 2492376
12  chromedriver                        0x000000010487894c chromedriver + 2492748
13  chromedriver                        0x0000000104888768 chromedriver + 2557800
14  libsystem_pthread.dylib             0x000000019c8de06c _pthread_start + 320
15  libsystem_pthread.dylib             0x000000019c8d8da0 thread_start + 8


2022-06-08 12:59:57,517 - ERROR - basePage.py[51]:用户名元素定位不到
2022-06-08 12:59:57,517 - ERROR - basePage.py[52]:Traceback (most recent call last):
  File "/Users/shenyuming/PycharmProjects/autotest_UI/common/basePage.py", line 45, in get_element
    return WebDriverWait(self.driver,ENV.WEB_DRIVER_WAIT_TIMEOUT,ENV.POLL_FREQUENCY_TIME).until(EC.visibility_of_element_located(locator))
  File "/usr/local/lib/python3.7/site-packages/selenium/webdriver/support/wait.py", line 89, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
0   chromedriver                        0x0000000104883da4 chromedriver + 2538916
1   chromedriver                        0x0000000104e19928 chromedriver + 8395048
2   chromedriver                        0x0000000104640e60 chromedriver + 167520
3   chromedriver                        0x000000010466fb48 chromedriver + 359240
4   chromedriver                        0x0000000104698740 chromedriver + 526144
5   chromedriver                        0x0000000104665158 chromedriver + 315736
6   chromedriver                        0x0000000104850cb0 chromedriver + 2329776
7   chromedriver                        0x0000000104861030 chromedriver + 2396208
8   chromedriver                        0x0000000104837f64 chromedriver + 2228068
9   chromedriver                        0x0000000104861fc4 chromedriver + 2400196
10  chromedriver                        0x0000000104847c5c chromedriver + 2292828
11  chromedriver                        0x00000001048787d8 chromedriver + 2492376
12  chromedriver                        0x000000010487894c chromedriver + 2492748
13  chromedriver                        0x0000000104888768 chromedriver + 2557800
14  libsystem_pthread.dylib             0x000000019c8de06c _pthread_start + 320
15  libsystem_pthread.dylib             0x000000019c8d8da0 thread_start + 8


2022-06-08 13:00:08,937 - ERROR - basePage.py[51]:用户名元素定位不到
2022-06-08 13:00:08,938 - ERROR - basePage.py[52]:Traceback (most recent call last):
  File "/Users/shenyuming/PycharmProjects/autotest_UI/common/basePage.py", line 45, in get_element
    return WebDriverWait(self.driver,ENV.WEB_DRIVER_WAIT_TIMEOUT,ENV.POLL_FREQUENCY_TIME).until(EC.visibility_of_element_located(locator))
  File "/usr/local/lib/python3.7/site-packages/selenium/webdriver/support/wait.py", line 89, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
0   chromedriver                        0x0000000104883da4 chromedriver + 2538916
1   chromedriver                        0x0000000104e19928 chromedriver + 8395048
2   chromedriver                        0x0000000104640e60 chromedriver + 167520
3   chromedriver                        0x000000010466fb48 chromedriver + 359240
4   chromedriver                        0x0000000104698740 chromedriver + 526144
5   chromedriver                        0x0000000104665158 chromedriver + 315736
6   chromedriver                        0x0000000104850cb0 chromedriver + 2329776
7   chromedriver                        0x0000000104861030 chromedriver + 2396208
8   chromedriver                        0x0000000104837f64 chromedriver + 2228068
9   chromedriver                        0x0000000104861fc4 chromedriver + 2400196
10  chromedriver                        0x0000000104847c5c chromedriver + 2292828
11  chromedriver                        0x00000001048787d8 chromedriver + 2492376
12  chromedriver                        0x000000010487894c chromedriver + 2492748
13  chromedriver                        0x0000000104888768 chromedriver + 2557800
14  libsystem_pthread.dylib             0x000000019c8de06c _pthread_start + 320
15  libsystem_pthread.dylib             0x000000019c8d8da0 thread_start + 8


2022-06-08 13:00:20,275 - ERROR - basePage.py[51]:用户名元素定位不到
2022-06-08 13:00:20,276 - ERROR - basePage.py[52]:Traceback (most recent call last):
  File "/Users/shenyuming/PycharmProjects/autotest_UI/common/basePage.py", line 45, in get_element
    return WebDriverWait(self.driver,ENV.WEB_DRIVER_WAIT_TIMEOUT,ENV.POLL_FREQUENCY_TIME).until(EC.visibility_of_element_located(locator))
  File "/usr/local/lib/python3.7/site-packages/selenium/webdriver/support/wait.py", line 89, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
0   chromedriver                        0x0000000104883da4 chromedriver + 2538916
1   chromedriver                        0x0000000104e19928 chromedriver + 8395048
2   chromedriver                        0x0000000104640e60 chromedriver + 167520
3   chromedriver                        0x000000010466fb48 chromedriver + 359240
4   chromedriver                        0x0000000104698740 chromedriver + 526144
5   chromedriver                        0x0000000104665158 chromedriver + 315736
6   chromedriver                        0x0000000104850cb0 chromedriver + 2329776
7   chromedriver                        0x0000000104861030 chromedriver + 2396208
8   chromedriver                        0x0000000104837f64 chromedriver + 2228068
9   chromedriver                        0x0000000104861fc4 chromedriver + 2400196
10  chromedriver                        0x0000000104847c5c chromedriver + 2292828
11  chromedriver                        0x00000001048787d8 chromedriver + 2492376
12  chromedriver                        0x000000010487894c chromedriver + 2492748
13  chromedriver                        0x0000000104888768 chromedriver + 2557800
14  libsystem_pthread.dylib             0x000000019c8de06c _pthread_start + 320
15  libsystem_pthread.dylib             0x000000019c8d8da0 thread_start + 8


2022-06-08 13:00:31,763 - ERROR - basePage.py[51]:用户名元素定位不到
2022-06-08 13:00:31,763 - ERROR - basePage.py[52]:Traceback (most recent call last):
  File "/Users/shenyuming/PycharmProjects/autotest_UI/common/basePage.py", line 45, in get_element
    return WebDriverWait(self.driver,ENV.WEB_DRIVER_WAIT_TIMEOUT,ENV.POLL_FREQUENCY_TIME).until(EC.visibility_of_element_located(locator))
  File "/usr/local/lib/python3.7/site-packages/selenium/webdriver/support/wait.py", line 89, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
0   chromedriver                        0x0000000104883da4 chromedriver + 2538916
1   chromedriver                        0x0000000104e19928 chromedriver + 8395048
2   chromedriver                        0x0000000104640e60 chromedriver + 167520
3   chromedriver                        0x000000010466fb48 chromedriver + 359240
4   chromedriver                        0x0000000104698740 chromedriver + 526144
5   chromedriver                        0x0000000104665158 chromedriver + 315736
6   chromedriver                        0x0000000104850cb0 chromedriver + 2329776
7   chromedriver                        0x0000000104861030 chromedriver + 2396208
8   chromedriver                        0x0000000104837f64 chromedriver + 2228068
9   chromedriver                        0x0000000104861fc4 chromedriver + 2400196
10  chromedriver                        0x0000000104847c5c chromedriver + 2292828
11  chromedriver                        0x00000001048787d8 chromedriver + 2492376
12  chromedriver                        0x000000010487894c chromedriver + 2492748
13  chromedriver                        0x0000000104888768 chromedriver + 2557800
14  libsystem_pthread.dylib             0x000000019c8de06c _pthread_start + 320
15  libsystem_pthread.dylib             0x000000019c8d8da0 thread_start + 8


2022-06-08 13:00:45,314 - ERROR - basePage.py[51]:用户名元素定位不到
2022-06-08 13:00:45,314 - ERROR - basePage.py[52]:Traceback (most recent call last):
  File "/Users/shenyuming/PycharmProjects/autotest_UI/common/basePage.py", line 45, in get_element
    return WebDriverWait(self.driver,ENV.WEB_DRIVER_WAIT_TIMEOUT,ENV.POLL_FREQUENCY_TIME).until(EC.visibility_of_element_located(locator))
  File "/usr/local/lib/python3.7/site-packages/selenium/webdriver/support/wait.py", line 89, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
0   chromedriver                        0x0000000104883da4 chromedriver + 2538916
1   chromedriver                        0x0000000104e19928 chromedriver + 8395048
2   chromedriver                        0x0000000104640e60 chromedriver + 167520
3   chromedriver                        0x000000010466fb48 chromedriver + 359240
4   chromedriver                        0x0000000104698740 chromedriver + 526144
5   chromedriver                        0x0000000104665158 chromedriver + 315736
6   chromedriver                        0x0000000104850cb0 chromedriver + 2329776
7   chromedriver                        0x0000000104861030 chromedriver + 2396208
8   chromedriver                        0x0000000104837f64 chromedriver + 2228068
9   chromedriver                        0x0000000104861fc4 chromedriver + 2400196
10  chromedriver                        0x0000000104847c5c chromedriver + 2292828
11  chromedriver                        0x00000001048787d8 chromedriver + 2492376
12  chromedriver                        0x000000010487894c chromedriver + 2492748
13  chromedriver                        0x0000000104888768 chromedriver + 2557800
14  libsystem_pthread.dylib             0x000000019c8de06c _pthread_start + 320
15  libsystem_pthread.dylib             0x000000019c8d8da0 thread_start + 8


2022-06-08 13:01:01,828 - ERROR - basePage.py[51]:用户名元素定位不到
2022-06-08 13:01:01,828 - ERROR - basePage.py[52]:Traceback (most recent call last):
  File "/Users/shenyuming/PycharmProjects/autotest_UI/common/basePage.py", line 45, in get_element
    return WebDriverWait(self.driver,ENV.WEB_DRIVER_WAIT_TIMEOUT,ENV.POLL_FREQUENCY_TIME).until(EC.visibility_of_element_located(locator))
  File "/usr/local/lib/python3.7/site-packages/selenium/webdriver/support/wait.py", line 89, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
0   chromedriver                        0x0000000104883da4 chromedriver + 2538916
1   chromedriver                        0x0000000104e19928 chromedriver + 8395048
2   chromedriver                        0x0000000104640e60 chromedriver + 167520
3   chromedriver                        0x000000010466fb48 chromedriver + 359240
4   chromedriver                        0x0000000104698740 chromedriver + 526144
5   chromedriver                        0x0000000104665158 chromedriver + 315736
6   chromedriver                        0x0000000104850cb0 chromedriver + 2329776
7   chromedriver                        0x0000000104861030 chromedriver + 2396208
8   chromedriver                        0x0000000104837f64 chromedriver + 2228068
9   chromedriver                        0x0000000104861fc4 chromedriver + 2400196
10  chromedriver                        0x0000000104847c5c chromedriver + 2292828
11  chromedriver                        0x00000001048787d8 chromedriver + 2492376
12  chromedriver                        0x000000010487894c chromedriver + 2492748
13  chromedriver                        0x0000000104888768 chromedriver + 2557800
14  libsystem_pthread.dylib             0x000000019c8de06c _pthread_start + 320
15  libsystem_pthread.dylib             0x000000019c8d8da0 thread_start + 8


2022-06-08 13:01:13,184 - ERROR - basePage.py[51]:用户名元素定位不到
2022-06-08 13:01:13,184 - ERROR - basePage.py[52]:Traceback (most recent call last):
  File "/Users/shenyuming/PycharmProjects/autotest_UI/common/basePage.py", line 45, in get_element
    return WebDriverWait(self.driver,ENV.WEB_DRIVER_WAIT_TIMEOUT,ENV.POLL_FREQUENCY_TIME).until(EC.visibility_of_element_located(locator))
  File "/usr/local/lib/python3.7/site-packages/selenium/webdriver/support/wait.py", line 89, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
0   chromedriver                        0x0000000104883da4 chromedriver + 2538916
1   chromedriver                        0x0000000104e19928 chromedriver + 8395048
2   chromedriver                        0x0000000104640e60 chromedriver + 167520
3   chromedriver                        0x000000010466fb48 chromedriver + 359240
4   chromedriver                        0x0000000104698740 chromedriver + 526144
5   chromedriver                        0x0000000104665158 chromedriver + 315736
6   chromedriver                        0x0000000104850cb0 chromedriver + 2329776
7   chromedriver                        0x0000000104861030 chromedriver + 2396208
8   chromedriver                        0x0000000104837f64 chromedriver + 2228068
9   chromedriver                        0x0000000104861fc4 chromedriver + 2400196
10  chromedriver                        0x0000000104847c5c chromedriver + 2292828
11  chromedriver                        0x00000001048787d8 chromedriver + 2492376
12  chromedriver                        0x000000010487894c chromedriver + 2492748
13  chromedriver                        0x0000000104888768 chromedriver + 2557800
14  libsystem_pthread.dylib             0x000000019c8de06c _pthread_start + 320
15  libsystem_pthread.dylib             0x000000019c8d8da0 thread_start + 8


